bin/update

```diff
@@ -28,18 +40,20 @@ chdir APP_ROOT do
   # If we need a "reset all the things" script, we should call it bin/reset.
   system! 'bundle update'
 
+  # load 'Rakefile'
+  load_rakefile
   puts "\n== Updating database =="
-  system! 'bin/rails db:migrate'
+  Rake::Task["db:migrate"].invoke
 
   puts "\n== Seeding database =="
-  system! "bin/rails db:seed GOOD_MIGRATIONS=skip"
+  Rake::Task["db:seed"].invoke
```

Comment:
So, I agree, it's wasteful to load environment several times (once per system call on bin/rails) but some rake tasks don't cleanup after themselves and cause subsequent rake tasks in the same process to fail.

See @chrisarcand's fix in https://github.com/rails/rails/pull/24305.

That's my concern here.